comtypes/test/test_findgendir.py

```diff
@@ -1,14 +1,17 @@
-import types, os, unittest, sys, tempfile
-import importlib
-
-if sys.version_info >= (2, 6):
-    from imp import reload
+import os
+import sys
+import tempfile
+import types
+import unittest
 
 import comtypes
 import comtypes.client
 import comtypes.gen
 
-from comtypes.client._code_cache import _get_appdata_dir
+if sys.version_info[0] == 3:
+    from importlib import reload
+elif sys.version_info[0] == 2:
+    from imp import reload
```

Comment:
`import.reload` is `New in version 3.4`.
https://docs.python.org/3.5/library/importlib.html#importlib.reload

To py33-compatible, use `from imp import reload`.
https://docs.python.org/3/library/imp.html#module-imp
```py
if sys.version_info >= (3, 4):
    from importlib import reload
else:
    from imp import reload
```

Reply:
Or we can drop Python 3.3 support in AppVeyor matrix.

Reply:
> Or we can drop Python 3.3 support in AppVeyor matrix.

It sounds better than changing this condition.

Reply:
However, it is premature to drop Python X.X.

I think that this PR(by @dmwyatt) goal is not dropping Python X.X from AppVeyor matrix and supports.

The goal as is it's title, `Make test suite runnable`.

We should discuss dropping versions in another scope, not in this PR.